dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5.1/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/krb5/CommonBindTest.java

```diff
@@ -98,24 +118,60 @@ public class CommonBindTest {
     protected static final String UNBOUNDID_USER_DN = "uid=" + UNBOUNDID_USER + "," + UNBOUNDID_BASE_DN;
     protected static final String UNBOUNDID_PWD = "usrpwd";
 
-    static int LDAP_PORT = ApacheDSandKDC.getLdapPort();
-    static int KDC_PORT = ApacheDSandKDC.getKdcPort();
+    static int LDAP_PORT = 389; //ApacheDSandKDC.getLdapPort();
+    static int KDC_PORT = 88; //ApacheDSandKDC.getKdcPort();
 
     protected static String[] stopStrings = null;
 
     @BeforeClass
     public static void setup() throws Exception {
-        ticketCacheFile = ApacheDSandKDC.getDefaultTicketCacheFile();
+        String usingRemoteDocker = System.getProperty("fat.test.use.remote.docker");
+        Log.info(c, "setUp", "Property fat.test.use.remote.docker: " + usingRemoteDocker);
+
+        // if the property is true or null(i.e. not false)
+        if (!"false".equals(usingRemoteDocker)) {
+            // Set the ldap hostname to the hostname of the remote docker host.
+            ldapServerHostName = ExternalDockerClientFilter.instance().getHostname();
+        } // else defaulted to: localhost
+        Log.info(c, "setUp", "setting ldap hostname to: " + ldapServerHostName);
```

Comment:
So this does not solve the problem I mentioned earlier. 

If `fat.test.use.remote.docker` is unset but we are running on an ARM mac we will still force the use of a remote docker host.  
Thus you will fail to enter this if block and default to a `localhost` even though your test is running on an external docker host.

Reply:
I see, another good catch